### PR TITLE
ref #23 - Upgrade django-chartjs to python35 and Django 1.8+

### PR DESCRIPTION
**Pull request by aRkadeFR**
### Upgrade support to python 2.7 + 3.4 + 3.5, on Django 1.8 + 1.9 and 1.10

ref #23 

- [x] change tox.ini
- [x] change .travis.yml
- [x] update demoproject and settings in order to unit tests to pass
- [ ] need to do a new release or release on PyPi for python3 ?

### CODE REVIEW

**Comment by aRkadeFR** [comment]
Should we release a new version or same version under universal wheel for py2 and py3?

**Comment by Natim** [comment]
A new version :)

**Comment by aRkadeFR** [comment]
PR ready for review

File: tox.ini [hunk 1]
```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py27,py33
+envlist = django{18,19,110}
```
Comment (zebuline) [inline tox.ini]: python version list not needed in tox.ini ? the definition in travis.yml is enough ?

Comment (aRkadeFR) [inline tox.ini]: I don't understand :/
We are building a python matrice in travis with python 2.7, 3.4, 3.5, and for each python version, we are running unittest through tox with Django versions: django 1.8, 1.9 and 1.10

Comment (k4nar) [inline tox.ini]: In most of our projects we let tox do the full matrix (python & django version), but both works :) .

Comment (aRkadeFR) [inline tox.ini]: Ah ok, I don't have any religion on this, I can change it if you want.

**Comment by Natim** [comment]
Yes I guess it is good if people can run the same things as travis does locally using `tox`

**Comment by aRkadeFR** [comment]
Updated to have the matrice in tox :)

File: tox.ini [hunk 2]
```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py27,py33
+envlist = django{18,19,110}-py{27,34,35}
 
 [testenv]
+deps =
+	django18: Django==1.8
```
Comment (zebuline) [inline tox.ini]: django18: Django>=1.8,<1.9
so you have the last 1.8.x

Comment (aRkadeFR) [inline tox.ini]: Updated

**Pull request merged**